yesod-auth/Yesod/Auth/Email.hs

```diff
@@ -188,6 +198,12 @@ class ( YesodAuth site
     -- @since 1.1.0
     addUnverified :: Email -> VerKey -> AuthHandler site (AuthEmailId site)
 
+    -- | Similar to `addUnverified`, but comes with the registered password
```

Comment:
Can you clarify what the default implementation is, and when you'd want to override it?